src/compas_view2/shapes/vectorgroup.py

```diff
@@ -0,0 +1,55 @@
+from compas.geometry import Vector
+from compas.geometry import Shape
+
+
+__all__ = ['VectorGroup']
+
+
+class VectorGroup(Shape):
+
+    def __init__(self, vectors, positions=None):
+        super().__init__()
+        print(vectors[0])
+        self.vectors = [Vector(*v) for v in vectors]
+        if positions:
+            self.positions = [Vector(*p) for p in positions]
+        else:
+            self.positions = [Vector(0, 0, 0) for _ in range(len(vectors))]
+        
```

Comment:
your linter must be on a break :)

Reply:
lol I thought it passed linter test, will double check